tests/components/valve/test_init.py

```diff
@@ -66,8 +66,30 @@ def __init__(
             self._attr_device_class = device_class
 
     def set_valve_position(self, position: int) -> None:
```

Comment:
Either change this to the async version or use `self.schedule_update_ha_state` below instead of `self.async_write_ha_state`.

Reply:
We should use the blocking version here to ensure full code coverage; changes to call `self.schedule_update_ha_state`